lib/Doctrine/DBAL/Platforms/SqlitePlatform.php

```diff
@@ -897,7 +897,7 @@ public function getAlterTableSQL(TableDiff $diff)
         if (! $this->onSchemaAlterTable($diff, $tableSql)) {
             $dataTable = new Table('__temp__' . $table->getName());
 
-            $newTable = new Table($table->getQuotedName($this), $columns, $this->getPrimaryIndexInAlteredTable($diff), [], $this->getForeignKeysInAlteredTable($diff), $table->getOptions());
+            $newTable = new Table($table->getQuotedName($this), array_values($columns), $this->getPrimaryIndexInAlteredTable($diff), [], $this->getForeignKeysInAlteredTable($diff), $table->getOptions());
```

Comment:
After switching from `array<int, Column>` to `array<int|string, Column>`, this is probably no longer needed.